app/models/post.rb

```diff
@@ -166,6 +166,10 @@ def self.find_by_detail(key, value)
     includes(:post_details).find_by(post_details: { key: key, value: value })
   end
 
+  def self.find_by_number(topic_id, post_number)
+    find_by(topic_id: topic_id, post_number: post_number)
+  end
+
```

Comment:
Is that helper used anywhere?

Reply:
No, neither are the other ones introduced here, they are all intended to make debugging this IMAP/email stuff way easier in `rails console`. This one for example turns `Topic.find(x).posts.find_by(post_number: y)` into `Post.find_by_number(x, y)`